### PR TITLE
docs cleaning - testcode

### PR DESCRIPTION
**Pull request by Borda**
## What does this PR do?

Fixes # (issue) <- this [links related issue to this PR](https://docs.github.com/en/free-pro-team@latest/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword)

## Before submitting
- [ ] Was this discussed/approved via a GitHub issue? (not for typos and docs)
- [x] Did you read the [contributor guideline](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/.github/CONTRIBUTING.md), **Pull Request** section?
- [x] Did you make sure your PR does only one thing, instead of bundling different changes together?
- [x] Did you make sure to update the documentation with your changes? (if necessary)
- [x] Did you write any new necessary tests? (not for typos and docs)
- [x] Did you verify new and existing tests pass locally with your changes?
- [ ] Did you update the [CHANGELOG](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/CHANGELOG.md)? (not for typos, docs, test updates, or internal minor changes/refactorings)

## PR review
Anyone in the community is free to review the PR once the tests have passed.
Before you start reviewing make sure you have read [Review guidelines](https://github.com/PyTorchLightning/pytorch-lightning/wiki/Review-guidelines). In short, see the following bullet-list:

 - [x] Is this pull request ready for review? (if not, please submit in draft mode)
 - [x] Check that all items from **Before submitting** are resolved
 - [x] Make sure the title is self-explanatory and the description concisely explains the PR
 - [x] Add labels and milestones (and optionally projects) to the PR so it can be classified
 - [x] **Check that target branch and milestone match!**

## Did you have fun?
Make sure you had fun coding 🙃

### CODE REVIEW

File: docs/source/introduction_guide.rst [hunk 1]
```diff
@@ -314,7 +314,7 @@ An alternative to using a DataModule is to defer initialization of the models mo
             # step is either 'fit' or 'test' 90% of the time not relevant
             data = load_data()
             num_classes = data.classes
-            self.l1 = nn.Linear(..., num_classes)
+            self.l1 = torch.nn.Linear(..., num_classes)
```
Comment (awaelchli) [inline docs/source/introduction_guide.rst:317]: can we keep it nn?
most pytorch examples also do import torch.nn as nn
we should just do the same at the top in the setup block

Comment (Borda) [reply docs/source/introduction_guide.rst]: done for all...

**Review by awaelchli — APPROVED** [review]
:)

**Comment by ydcjeff** [comment]
Docstring is not rendering correctly.
https://90378-178626720-gh.circle-artifacts.com/0/html/plugins.html#pytorch_lightning.plugins.apex.ApexPlugin.clip_gradients

**Comment by Borda** [comment]
> Docstring is not rendering correctly.

good catch, just for. the record this was there even before, I have not touched this section :]

**Review by rohitgr7 — APPROVED** [review]
nice cleanup!!

**Review by carmocca — APPROVED** [review]

**Review by tchaton — APPROVED** [review]

**Review by williamFalcon — APPROVED** [review]
